src/components/CytoscapeGraph/CytoscapeContextMenu.tsx

```diff
@@ -79,7 +79,7 @@ export class CytoscapeContextMenuWrapper extends React.PureComponent<Props> {
       event.preventDefault();
       if (event.target) {
         const currentContextMenu = this.getCurrentContextMenu();
-        if (currentContextMenu) {
+        if (!!currentContextMenu) {
```

Comment:
Nitpick: I think this double negation is not needed.

Anyway, this same condition is on other places. If this is really contributing to the fix, I think the other places should also be changed.